scenarios/README.md

```diff
@@ -0,0 +1,50 @@
+# Scenarios
+
+This folder contains various projects conducted in cyclus.
+
+## 2014-E&S
```

Comment:
I would see if you can add a link to the E&S website.

Reply:
Some of this work may have also ended up in Gwen's MS thesis.

Reply:
I'll check it out!